### ISSUE TITLE
What is `@codecs` for metadata adaptation set supposed to be?

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#290]
I am unable to decode the following text in IOP:

> Metadata Adaptation Set: An Adaptation Set that contains information that is not expected to be rendered by a specific media handler, but is interpreted by the application. Such an Adaptation Set is identified by @mimeType="application/mp4" and an appropriate sample entry identified by the @codecs parameter.

What is "an appropriate sample entry"? Does it refer to something (what?) in the media coding technologies section? If I am a client, how do I determine whether an adaptation set is a metadata adaptation set? We need a clearer definition here.

**Comment by RufaelDev** [comment]
how about 'meta' and allowing extensions meta.XXX ?

**Comment by haudiobe** [comment]
I agree that it is tricky, but this is what it is. We have no clear signaling for metadata. If people want to solve it, please bring it to IETF and MPEG, and do not solve it in DASH-IF.

**Comment by sandersaares** [comment]
IOP mentions "metadata adaptation set" in exactly 1 place: when defining it.

Considering the above comment that it is tricky to even define, I propose we eliminate this concept from IOP. If we do not use this definition, we have no need to define it.
